fasm2bels/lib/interchange_capnp.py

```diff
@@ -24,11 +24,61 @@
 import capnp
 import capnp.lib.capnp
 capnp.remove_import_hook()
+import gzip
 import os.path
 from .logical_netlist import check_logical_netlist, Library, Cell, Direction
 from .physical_netlist import Placement, PhysicalPip, stitch_stubs
 from ..models.verilog_modeling import make_bus, flatten_wires, unescape_verilog_name
 
+# Flag indicating use of Packed Cap'n Proto Serialization
+IS_PACKED = False
+# Flag indicating that files are gziped on output
+IS_GZIPPED = True
+
+
+def read_capnp_file(capnp_schema,
+                    f_in,
+                    is_gzipped=IS_GZIPPED,
```

Comment:
`is_gziped=` would be better as `compression_format=`?

Reply:
Sure